config/config.go

```diff
@@ -357,6 +360,7 @@ type Configuration struct {
 	Mining                       bool          `json:"Mining"`
 	MiningDebug                  bool          `json:"MiningDebug"`
 	BeneficiaryAddr              string        `json:"BeneficiaryAddr"`
+	SyncStateMaxPeer             uint32        `json:"SyncStateMaxPeer"`
```

Comment:
Better rename to `SyncStateMaxWorkers` because `concurrentSyncRequestPerNeighbor` might be more than 1 in the future

Reply:
Fixed